src/api/index.ts

```diff
@@ -148,7 +149,6 @@ router.get(
 	"/:installationId/repoSyncState.json",
 	check("installationId").isInt(),
 	returnOnValidationError,
-	elapsedTimeMetrics,
```

Comment:
Removed metrics from individual calls and put them on the router level because:
* If they are here it wouldn't capture 401s and other validation error responses
* On this level It wouldn't include the validation and authentication times

Reply:
Nice!